frontend/package.json

```diff
@@ -7,8 +7,11 @@
     "query-string": "^6.7.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-dropzone": "^10.1.5",
+    "react-forms-validator": "^1.0.8",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
+    "superagent": "^5.1.0"
```

Comment:
what is this dependency for?

Reply:
Must be an old dependency that I never uninstalled.